packages/beacon-types/src/types/Regions.ts

```diff
@@ -0,0 +1,9 @@
+export enum Regions {
+  EU1 = 'eu-1',
+  US1 = 'us-1'
+}
+
+export interface NodeDistributions {
+  [Regions.EU1]?: string[]
```

Comment:
@AndreasGassmann after thinking it through, I would opt for allowing users to define their own regions, because, simply, why shouldn't anyone be able to provide nodes based in Asia or Australia and name them accordingly, without asking you to add the values in the enum?